raco/algebra.py

```diff
@@ -1104,6 +1161,9 @@ def num_tuples(self):
     def shortStr(self):
         return self.opname()
 
+    def partitioning(self):
+        return self.input.partitioning()
+
 
 class PartitionBy(UnaryOperator):
```

Comment:
I interpreted Split to be a local operation (no communication cross-worker) used for splitting a fragment in two pieces. Is this not the case?